rest-proxy-boot/build.gradle

```diff
@@ -10,7 +10,8 @@ buildscript {
         mavenCentral()
     }
     dependencies {
-        classpath("org.springframework.boot:spring-boot-gradle-plugin:1.3.6.RELEASE")
+        classpath("org.springframework.boot:spring-boot-gradle-plugin:1.3.7.RELEASE")
+        classpath "io.spring.gradle:dependency-management-plugin:0.6.1.RELEASE"
```

Comment:
Oh yeah. It's necessary to work with Gradle 3, as the spring boot gradle plugin requires a class called StyledTextOutputFactory.

See https://github.com/spring-gradle-plugins/dependency-management-plugin/issues/87